src/components/AnalysisPage/ThicknessAnalysisBox/StartButton/StartButton.jsx

```diff
@@ -71,11 +74,19 @@ const StartButton = ({
   }
 
   function isValidThresholdInput(threshold) {
-    if (threshold === "." || threshold === ".0" || threshold === "" || threshold === "0" || threshold === "0.0" || threshold === '0.' || threshold === null) {
-      return false
+    if (
+      threshold === "." ||
+      threshold === ".0" ||
+      threshold === "" ||
+      threshold === "0" ||
+      threshold === "0.0" ||
```

Comment:
Totally unrelated to this PR but I just realized I could probably also add '0.00' and '.00' to this list as well so that the button isnt clickable for that value